hws/src/dr_action.py

```diff
@@ -7,6 +7,14 @@
 from src.dr_hl import dr_hl_dw_parser
```

Comment:
Can't comment in the commit message (Github...), so I'm leaving the comment here: please fix 'CX6' and 'CX7' to 'ConnectX-6' and 'ConnectX-6' - protect the branding.

Reply:
Done